components/dashboards-web-component/webpack.config.js

```diff
@@ -55,14 +56,23 @@ module.exports = {
                     loader: 'sass-loader',
                 }],
             },
-
+            // {
+            //     test: /\.jsx?$/,
```

Comment:
Why disable es-lint?

Reply:
This is to enforce ESLint with webpack build. Unless we fix all the ESLint error we are not in a position to enforce this with the build. That's why I commented that out.